charts/vector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: vector
-version: "0.16.3"
+version: "0.16.4"
```

Comment:
Why is this necessary?

Reply:
Ah.. has this leaked in from the colour codes pr?

Reply:
We'd fail the check [here](https://github.com/vectordotdev/helm-charts/actions/runs/3282096145/jobs/5405027736#step:5:29).

The datadog agent does similarly, https://github.com/DataDog/helm-charts/pull/681